app/javascript/dashboard/routes/dashboard/conversation/ConversationView.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <section class="app-content columns">
-    <chat-list :conversation-inbox="inboxId" :label="label">
+    <chat-list
+      :conversation-inbox="inboxId"
+      :label="label"
+      @conversation-load="onConversationLoad"
```

Comment:
Good pattern 🔥 Should we have a comment to link about how events work in vue ?

Reply:
I guess it would be good to set a style guide website for this content.